src/common/Thread.h

```diff
@@ -61,6 +64,9 @@ class Thread {
   int join(void **prval = 0);
   int detach();
   int set_affinity(int cpuid);
+  static const std::string get_thread_name() {
+    return Thread::thread_name;
```

Comment:
There are many threads in Ceph (and the MDS) which do not use this `class Thread`. So the "name" will wrongly be empty every time this is called.

Reply:
It might be a good idea to make those threads use `Thread` class then.

Reply:
I think the idea behind helpers like `make_named_thread()` was the opposite -- slowly move off of `Thread` and just use `std::thread`.